pyproject.toml

```diff
@@ -38,6 +38,7 @@ dependencies = [
   "youtube-transcript-api",
   "SpeechRecognition",
   "pathvalidate",
+  "onenote",
```

Comment:
is this the correct package name? could not find it in pip package, but found this one instead https://pypi.org/project/one-extract/